ci-oiio/Dockerfile

```diff
@@ -0,0 +1,160 @@
+# syntax=docker/dockerfile:1.4
+# Copyright (c) Contributors to the aswf-docker Project. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# !!! This file is automatically generated from a template and the `image.yaml` file in the same folder !!!
+
+ARG ASWF_ORG
+ARG ASWF_PKG_ORG
+ARG CI_COMMON_VERSION
+ARG ASWF_CLANG_MAJOR_VERSION
+ARG ASWF_CONAN_CHANNEL
+
+ARG ASWF_VFXPLATFORM_VERSION
+
+
+ARG ASWF_BLOSC_VERSION
+ARG ASWF_BOOST_VERSION
+ARG ASWF_CMAKE_VERSION
+ARG ASWF_PYTHON_VERSION
+ARG ASWF_IMATH_VERSION
+ARG ASWF_OCIO_VERSION
+ARG ASWF_OPENEXR_VERSION
+ARG ASWF_OPENVDB_VERSION
+ARG ASWF_PARTIO_VERSION
```

Comment:
FWIW, OIIO doesn't use partio

Reply:
I probably based the ci-oiio container on the ci-osl one which has partio...

Will remove it in next version, thanks for catching that!